### PR TITLE
Docs: Release guide

### PR DESCRIPTION
**Pull request by moneymanolis**
This adds a release guide to help remembering all the necessary steps, especially since not everything is / can be automated.

### CODE REVIEW

File: docs/release-guide.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+### Creating release notes
+#### Pre-requisites
+- You need the correct upstream master. You should see
+```bash 
+upstream  git@github.com:cryptoadvance/specter-desktop.git
+```
+when you run
+```bash
+git remote -v
+```
```
Comment (k9ert) [inline docs/release-guide.md:10]:
```suggestion
```bash 
git remote -v | grep upstream                                                                  
upstream        git@github.com:cryptoadvance/specter-desktop.git (fetch)
upstream        git@github.com:cryptoadvance/specter-desktop.git (push)
```
```

File: docs/release-guide.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+### Creating release notes
```
Comment (k9ert) [inline docs/release-guide.md:1]:
```suggestion
# Release Guide

## Creating release notes
```

File: docs/release-guide.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Using the new token, run
+ ```bash
+ export GH_TOKEN=YOURTOKEN
+ ```
+- Checkout the master branch and ensure a clean workspace.
+
+Now, you can run
+```bash
+./utils/release.sh --release-notes
+```
```
Comment (k9ert) [inline docs/release-guide.md:25]: docker as a prerequisite is missing, no?!

File: docs/release-guide.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+There are three stages:
+![](./images/release-guide/overview-gitlab-pipline.png)
+
+The first relevant stage is "releasing". Here, the Windows, Linux and pip release are created and uploaded to the Specter Desktop GitHub releases page. After this stage, the following artificats should be available:
+
+cryptoadvance.specter-1.13.1.tar.gz
+Specter-Setup-v1.13.1.exe
+specterd-v1.13.1-win64.zip
+specterd-v1.13.1-x86_64-linux-gnu.zip
+specter_desktop-v1.13.1-x86_64-linux-gnu.tar.gz
```
Comment (k9ert) [inline docs/release-guide.md:49]:
```suggestion
- cryptoadvance.specter-1.13.1.tar.gz
- Specter-Setup-v1.13.1.exe
- specterd-v1.13.1-win64.zip
- specterd-v1.13.1-x86_64-linux-gnu.zip
- specter_desktop-v1.13.1-x86_64-linux-gnu.tar.gz
```

File: docs/release-guide.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+The three jobs in more detail:
+release_binary_windows: is creating a binary for specterd and for Windows (Windows runner)
+release_electron_linux_windows: Creates a specterd for Linux, an AppImage for Linux and an executable for Windows (Linux runner).
+release_pip: Is releasing a pypi package on [pypi](https://pypi.org/project/cryptoadvance.specter/) and creates a tarball of the pip package for the GitHub release page (Linux runner).
```
Comment (k9ert) [inline docs/release-guide.md:54]:
```suggestion
- release_binary_windows: is creating a binary for specterd and for Windows (Windows runner)
- release_electron_linux_windows: Creates a specterd for Linux, an AppImage for Linux and an executable for Windows (Linux runner).
- release_pip: Is releasing a pypi package on [pypi](https://pypi.org/project/cryptoadvance.specter/) and creates a tarball of the pip package for the GitHub release page (Linux runner). 
```

File: docs/release-guide.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+specterd-v1.13.1-osx.zip
+SHA256SUMS-macos
+SHA256SUMS-macos.asc
+
+### GitLab - post releasing
+Back to GitLab, the final stage is "post releasing". 
+
+In this stage, the invididual SHA256-hashes and signatures are combined into two final files:
+SHA256SUMS
+SHA256SUMS.asc
```
Comment (k9ert) [inline docs/release-guide.md:76]:
```suggestion
- SHA256SUMS
- SHA256SUMS.asc
```

File: docs/release-guide.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+Follow the instructions in `README.md`
+`./build.sh`
+
+The result `gh_page.md` can then be found in the build directory.
+Edit the release on GitHub and paste this md-file there.
+
+### Website 
+The above script also produces html files for the website (in the same directory):
+`download-page_current.html`
+`download-page_releases.html`
```
Comment (k9ert) [inline docs/release-guide.md:111]:
```suggestion
- `download-page_current.html`
- `download-page_releases.html`
```

**Review by k9ert — CHANGES_REQUESTED** [review]
Some recommended changes ...

**Comment by moneymanolis** [comment]
All requests should have been addressed.

**Review by k9ert — APPROVED** [review]

**Pull request merged**